ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_random.py

```diff
@@ -270,5 +271,32 @@ def test_paddle_randint_like(
         x=x[0],
         low=low,
         high=high,
-        dtype=dtype[0]
-    )
\ No newline at end of file
+        dtype=dtype[0],
+    )
+
+
+# multinomial
+@handle_frontend_test(
+    fn_tree="paddle.tensor.random.multinomial",
+    dtype=helpers.get_dtypes("float", full=False),
+    n=helpers.ints(min_value=2, max_value=10),
+)
+def test_paddle_multinomial(
+    dtype,
+    n,
+    test_flags,
+    frontend,
+    backend_fw,
+    fn_tree,
+    on_device,
+):
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        test_flags=test_flags,
+        frontend=frontend,
+        backend_to_test=backend_fw,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        test_values=False,
+        x=np.array([1 / n] * n, dtype=dtype[0]),
```

Comment:
Why do we need to do this here and not generate this the usual way using `helpers.dtype_and_values` ?

Reply:
Basically I think you can give any array to paddle and it normalizes the probability array so that it sums up to 1 but for other frameworks like numpy we need to give it an array so that the probability array sums up to 1. Using `dtype_and_values` I don't think there is a way to make an array that is already normalized. Let me know what you think.

Reply:
We could do it like setting `min_value=0.25, max_value=0.25, min_dim_size=5, max_dim_size=5` in `dtype_and_values` so that the array always sums to 1. What do you think about this? Should be sufficient in order to test the function I presume.

Reply:
> We could do it like setting `min_value=0.25, max_value=0.25, min_dim_size=5, max_dim_size=4` in `dtype_and_values` so that the array always sums to 1. What do you think about this? Should be sufficient in order to test the function I presume.

@asad0991 I assume you meant `min_dim_size=4` here. If so, yeah I think this could work. Of course if it doesn't, feel free to create a composite strategy and taking the `np.array([1 / n] * n, dtype=dtype[0])` calculation inside that strategy. We can only put strategies here and not something like `x=np.array(...)` if that makes sense?